Assets/AnkrSDK/Plugins/WalletConnect.VersionShared/Models/Ethereum/EthChain.cs

```diff
@@ -0,0 +1,7 @@
+namespace AnkrSDK.Plugins.WalletConnect.VersionShared.Models.Ethereum
+{
+	public class EthChain
+	{
+		public string chainId;
```

Comment:
ChainId? If it is a serializable class mark it with the correct attribute.

Reply:
This class is the same as in master, it was just moved to a different assembly in order to be shared between WC and WC2